### ISSUE TITLE
Error message (9)

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#256]
Code:

```
"intParam"  : 1,
"dictParam" : {"A" : 1},
"param"     : ${intParam}[${dictParam}]
```

Result:

`Error: 'Invalid index or dictionary key in parameter '${intParam}[${dictParam}]'. The datatype of variable '${dictParam}' have to 'int' or 'str'.'!`

That's wrong. The data type of `${dictParam}` is irrelevant. Because of `${intParam}` is of data type `int`. And integers cannot have an index or a key.

The same in pure Python:

```
intParam  = 1
dictParam = {"A" : 1}
param     = intParam[dictParam]
```

causes:

`TypeError: 'int' object is not subscriptable`

Either you take over this original error message from Python interpreter or you provide an own interpretation, something like:

`TypeError: Integers cannot have an index or a key.`

**Comment by namsonx** [comment]
Hello Holger, 

I had updated on stabi branch, the error message now is ```Could not resolve expression '${intParam}[${dictParam}]'. Reason: 'int' object is not subscriptable```

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
released with RobotFramework AIO 0.13.0
